lib/DDG/Spice/Github.pm

```diff
@@ -16,10 +16,16 @@ triggers startend => "github";
 spice to => 'https://api.github.com/legacy/repos/search/$1?callback={{callback}}';
 spice proxy_cache_valid => '200 30d';
 
-handle query_lc => sub {
-    s/^github\s+|\s+github$//;
+handle remainder_lc => sub {
+
+	s/^github\s+|\s+github$//;
+	return unless $_;
     return $_ unless /^jobs\b|\bjobs$|^status\b|\bstatus$/;
-    return;
+   	return;
+	
+	
+	return;
+    
```

Comment:
Can you indent all code in the `handle` function using 4 spaces (soft tabs) as per the [code styleguide](https://duck.co/duckduckhack/code_styleguide)